oraichain/assetlist.json

```diff
@@ -50,6 +50,516 @@
+          "exponent": 0
+        },
+        {
+          "denom": "SCATOM",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "cw20",
+      "address": "orai19q4qak2g3cj2xc2y3060t0quzn3gfhzx08rjlrdd3vqxhjtat0cq668phq",
+      "base": "cw20:orai19q4qak2g3cj2xc2y3060t0quzn3gfhzx08rjlrdd3vqxhjtat0cq668phq",
+      "name": "scATOM",
+      "display": "scatom",
+      "coingecko_id": "scatom",
+      "symbol": "scATOM",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/scatom.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/scatom.svg"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/scatom.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/scatom.svg"
+      }
+    },
+    {
+      "description": "The xOCH token is the cw20 token for the Oraichain.",
+      "denom_units": [
+        {
+          "denom": "cw20:orai1lplapmgqnelqn253stz6kmvm3ulgdaytn89a8mz9y85xq8wd684s6xl3lt",
+          "exponent": 0
+        },
+        {
+          "denom": "XOCH",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "cw20",
+      "address": "orai1lplapmgqnelqn253stz6kmvm3ulgdaytn89a8mz9y85xq8wd684s6xl3lt",
+      "base": "cw20:orai1lplapmgqnelqn253stz6kmvm3ulgdaytn89a8mz9y85xq8wd684s6xl3lt",
+      "name": "Xoch",
+      "display": "xoch",
+      "coingecko_id": "och",
+      "symbol": "xOCH",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/och.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/och.svg"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/och.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/och.svg"
+      }
+    },
+    {
+      "description": "The wETH token is the cw20 token for the Oraichain.",
+      "denom_units": [
+        {
+          "denom": "cw20:orai1dqa52a7hxxuv8ghe7q5v0s36ra0cthea960q2cukznleqhk0wpnshfegez",
+          "exponent": 0
+        },
+        {
+          "denom": "WETH",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "cw20",
+      "address": "orai1dqa52a7hxxuv8ghe7q5v0s36ra0cthea960q2cukznleqhk0wpnshfegez",
+      "base": "cw20:orai1dqa52a7hxxuv8ghe7q5v0s36ra0cthea960q2cukznleqhk0wpnshfegez",
+      "name": "Weth",
+      "display": "weth",
+      "coingecko_id": "weth",
+      "symbol": "wETH",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/eth-black.png",
```

Comment:
this image URL doesn't lead to any image.

Reply:
> this image URL doesn't lead to any image.

I think when merge the code into the `master` branch, this image will work.

Reply:
Oh, yes you are trying to upload those images here.

Reply:
Please instead upload these to the ethereum images directory and register them as an additional image of the ETH asset on the ethereum assetlist